### PR TITLE
Refactor/remove pyxdg

### PR DESCRIPTION
**Pull request by NeonJarbas**
needs https://github.com/OpenVoiceOS/ovos_utils/pull/21

together with #52 closes https://github.com/OpenVoiceOS/ovos-core/issues/51

other repos also need this change
- [x] ovos_utils https://github.com/OpenVoiceOS/ovos_utils/pull/21
- [x] ovos plugin manager https://github.com/OpenVoiceOS/OVOS-plugin-manager/pull/30
- [x] ovos precise lite plugin https://github.com/OpenVoiceOS/ovos-ww-plugin-precise-lite/pull/1
- [ ] ....
- [x] json_database https://github.com/OpenJarbas/json_database/pull/1

### CODE REVIEW

**Comment by JarbasAl** [comment]
added workflow to validate license in every PR https://github.com/NeonJarbas/ovos-core/runs/5188268536?check_suite_focus=true

File: mycroft/skills/event_scheduler.py [hunk 1]
```diff
@@ -67,13 +66,12 @@ def __init__(self, bus, schedule_file='schedule.json', autostart=True):
...
         core_conf = Configuration.get(remote=False)
-        data_dir = core_conf.get('data_dir', xdg.BaseDirectory.save_data_path(get_xdg_base()))
+        data_dir = core_conf.get('data_dir') or get_xdg_data_save_path()
         old_schedule_path = join(expanduser(data_dir), schedule_file)
 
         self.schedule_file = old_schedule_path
         if is_using_xdg():
-            new_schedule_path = join(
-                xdg.BaseDirectory.load_first_config(get_xdg_base()), schedule_file)
+            new_schedule_path = join(get_xdg_data_save_path(), schedule_file)
```
[the remark this answers is not in the document]
Comment (NeonDaniel) [reply mycroft/skills/event_scheduler.py:74]: My inclination would be cache, since it is not really portable and is tied to a particular instance of core

**Review by NeonDaniel — COMMENTED** [review]
Need to test with updated ovos-utils. left one reply, otherwise changes look okay

**Comment by JarbasAl** [comment]
closes #51 

see https://github.com/NeonJarbas/ovos-core/runs/5203358862?check_suite_focus=true

**Pull request merged**